Assets/MixedRealityToolkit/Utilities/Scenes/RuntimeSceneUtils.cs

```diff
@@ -48,6 +48,11 @@ public static IEnumerable<GameObject> GetRootGameObjectsInLoadedScenes()
             for (int i = 0; i < SceneManager.sceneCount; i++)
             {
                 Scene loadedScene = SceneManager.GetSceneAt(i);
+                if (!loadedScene.isLoaded)
+                {   // Oh, Unity.
```

Comment:
Comment here is not needed

Reply:
i just copy/pasted the comment from the EditorSceneUtils version.
Thing is, there is a unity bug here, as SceneManager.sceneCount is supposed to only returned the loaded scenes, so we expect to have only scenes with isLoaded == true, which is not the case :p

Reply:
Instead of removing it why don't we replace it with something more descriptive. The point was to call out that the check really shouldn't be necessary.